networks/rpc/types.go

```diff
@@ -34,12 +34,26 @@ import (
 	"gopkg.in/fatih/set.v0"
 )
 
+type ServerScheme uint8
+
+const (
+	HTTPServer ServerScheme = iota
+	FastHTTPServer
+	WSServer
+	FastWSServer
+	GRPCServer
+	InProcServer
+	IPCServer
+	TestServer
+)
+
 // API describes the set of methods offered over the RPC interface
 type API struct {
-	Namespace string      // namespace under which the rpc methods of Service are exposed
-	Version   string      // api version for DApp's
-	Service   interface{} // receiver instance which holds the methods
-	Public    bool        // indication if the methods must be considered safe for public use
+	Namespace  string      // namespace under which the rpc methods of Service are exposed
+	Version    string      // api version for DApp's
+	Service    interface{} // receiver instance which holds the methods
+	Public     bool        // indication if the methods must be considered safe for public use
+	Privileged bool
```

Comment:
If you could comment simply what privilieged means, it would be nice for people looking at the code later.

Reply:
Oops, Thank you. Added